mmv1/third_party/terraform/tpgresource/common_diff_suppress.go

```diff
@@ -101,6 +101,13 @@ func ProjectNumberDiffSuppress(_, old, new string, _ *schema.ResourceData) bool
 	return a2 == b2
 }
 
+func SuppressRoutingModeDefault(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
can you add a comment explaining the purpose of this diff? I understand that this comes from this [comment](https://github.com/hashicorp/terraform-provider-google/issues/19826#issuecomment-3020709192) left on the issue but it would be helpful to have it be included with the definition of the diff function

Reply:
theres a formatting issue in the latest commits

https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/17106382056/job/48516276830?pr=14767